web/modules/custom/dpl_webform/dpl_webform.install

```diff
@@ -55,3 +55,21 @@ function dpl_webform_update_10002(): string {
 
   return 'All translations for the webform ' . $webform_id . ' have been deleted.';
 }
+
+/**
+ * Delete translation of webform.settings.yml.
+ *
+ * Due to some faulty, prior translation, the danish version of this file
+ * has duplicate keys, which causes major issues on the site when loading
+ * the webform settings.
+ */
+function dpl_webform_update_10004(): string {
+  $config_id = 'webform.settings';
+  $language = 'da';
+
+  /** @var \Drupal\language\Config\LanguageConfigFactoryOverride $language_config_factory_override */
+  $language_config_factory_override = \Drupal::service('language.config_factory_override');
+  $language_config_factory_override->getOverride($language, $config_id)->delete();
```

Comment:
I do not know if it makes sense to create a reusable function to address this. Perhaps next time.

Reply:
I mean, hopefully there won't be a next time, because it should never be necessary to do this